src/doc/DocDrawingDialog.js

```diff
@@ -164,6 +164,10 @@ class DocDrawingDialog extends AnnotationDialog {
             this.pageEl = this.annotatedElement.querySelector(`[data-page-number="${this.location.page}"]`);
         }
 
+        if (!this.element) {
+            this.setup([]);
```

Comment:
Kinda weird that we have to pass in an empty array here

Reply:
👍 Fixed